src/webapp/azext_webapp/__init__.py

```diff
@@ -51,14 +52,26 @@ def load_arguments(self, _):
             c.argument('dryrun', help="show summary of the create and deploy operation instead of executing it", default=False, action='store_true')
             c.argument('registry_rg', help="the resource group of the Azure Container Registry")
             c.argument('registry_name', help="the name of the Azure Container Registry")
+            c.argument('slot', help="the deployment slot")
```

Comment:
Maybe you need to make the help messages consistent.
In the other two places, you use
```
Name of the deployment slot to use
```